bitnami/solr/templates/ingress.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.ingress.enabled }}
```

Comment:
You are also missing a file like this: bitnami/nats/templates/tls-secret.yaml

This is the one that uses the value ingress.secrets